less/variables.less

```diff
@@ -117,6 +117,7 @@
 
 @buttonShadow: 0px 1px 5px -1px rgba(0, 0, 0, 0.5);
 @dialogShadow: 0px 8px 22px 0px rgba(0, 0, 0, .5);
+@flyoutDialogBoxShadow: 2px 2px 8px #3b3b3b;
```

Comment:
To echo my earlier comment; I don't think we should use non-black HEX colors for shadows. `rgba` handles this much better.